guides/v2.0/frontend-dev-guide/theme-best-practice.md

```diff
@@ -15,7 +15,7 @@ These approaches give higher chances to avoid conflicts and issues with your the
 
 ## Theme development best practice
 
-1. When [inheriting]({{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html) from a default Magento theme, rather extend, than override the default styles: whenever possible, put your customizations in the `_extend.less` or `_theme.less` file, instead of overriding a `.less` file from a parent theme. 
+1. When [inheriting]({{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html) from a default Magento theme, extend rather than override the default styles, whenever possible, Put your customizations in the `_extend.less` or `_theme.less` file, instead of overriding a `.less` file from a parent theme. 
```

Comment:
Missing the period after "possible" instead of the comma. The changes were applied inconsistently across versions.

Reply:
I updated the files to your requested changes.

Reply:
Thank you! 🦊